### PR TITLE
Bugfix/text widget overflow changes

### PR DESCRIPTION
**Pull request by Southeaven**

### CODE REVIEW

File: cogboard-webapp/src/components/widgets/types/TextWidget/index.js [hunk 1]
```diff
@@ -1,16 +1,53 @@
...
+import {
+  TypographyVariant,
+  CenterWrapper,
+  StyledPre,
+  RotatedStyledPre,
+  OverflowingText,
+  SetWidth
+} from './styled';
+
+export const ModifiedWidth = (component, height) => {
```
Comment (marcin-witaszek) [inline cogboard-webapp/src/components/widgets/types/TextWidget/index.js]: Imho helper functions/hooks names should start with lowercase

Comment (marcin-witaszek) [reply cogboard-webapp/src/components/widgets/types/TextWidget/index.js]: Sorry, now I realised that these are subcomponents - maybe their names are a bit misleading

Comment (Southeaven) [reply cogboard-webapp/src/components/widgets/types/TextWidget/index.js]: What names would fit them better in your opinion?

Comment (marcin-witaszek) [reply cogboard-webapp/src/components/widgets/types/TextWidget/index.js]: Maybe turn it into some kind of HOC and add `width` prefix... I don't have a clear idea for these names - for me they are totally ok

**Pull request merged**